src/pg_patterns.cpp

```diff
@@ -3258,6 +3258,22 @@ Pattern *ImportPatternUI::get_pattern()
 	return pattern;
 }
 
+void ImportPatternUI::reloadFileData()
+{
+	loadFileData();
+	parse_ui();
+}
+
+void ImportPatternUI::loadFileData()
+{
+
+	FileManager fm("Pattern Generator");
+	fm.open(pattern->fileName, FileManager::IMPORT);
```

Comment:
fm.open() might throw an exception if the file is corrupt, or it can't be opened. There should be a try-catch block here to make sure Scopy won't crash when this happens.

Reply:
Fixed